MyOwnORM/MyOwnORM/CustomDbContextOptionsBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbContextOptionsBuilder : CustonSqlServerContextOptionsBuilderExtension
```

Comment:
why do you need this class

Reply:
After first review I think that this class hasn't anything useful and I deleted 'CustonSqlServerContextOptionsBuilderExtension' and enter him logic into 'CustomDbContextOptionsBuilder', because I thought that ORM should work with differents DBMS and I will create another classes 'CustonPostgresSqlContextOptionsBuilderExtension' or 'CustonMySqlContextOptionsBuilderExtension'